src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
-body {
-  margin: 0;
-}
+@import './block/@media';
+@import './block/*';
+@import './block/html';
+@import './block/body';
+@import './block//header';
```

Comment:
There is a typo in the import path for the header block. The path './block//header' should be corrected to './block/header' to ensure the styles are properly imported.